sale_order_revision/view/sale_order.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
```

Comment:
Make XML record ID unique, 
```suggestion
    <record id="view_order_form_inherit_revision" model="ir.ui.view">
```

Reply:
I think this one is ok. The full external ID will be `sale_order_revision.view_order_form` anyway.